examples/framework.py

```diff
@@ -116,7 +125,10 @@ def configure(self):
         if self.args.ask_password:
             password = getpass.getpass()
         else:
-            password = self.config.get("global", "password")
+            if self.config.has_section(self.g_jid):
+                password = self.config.get(self.g_jid, 'password')
```

Comment:
```
horazont@sinistra tmp › cat foo.ini
[foo@bar.baz]
test=test

horazont@sinistra tmp › python3
Python 3.5.2+ (default, Sep 22 2016, 12:18:14) 
[GCC 6.2.0 20160927] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import configparser
>>> p = configparser.ConfigParser()
>>> p.read("foo.ini")
['foo.ini']
>>> p.has_section("foo@bar.baz")
True
>>> import aioxmpp
>>> j = aioxmpp.JID.fromstr("foo@bar.baz")
>>> str(j) == "foo@bar.baz"
True
>>> p.has_section(j)
False
>>> p.has_section(str(j))
True
>>> 

```

It appears that has_section will always return false when passed a JID instance.